src/appmixer/quickbooks/accounting/NewInvoice/NewInvoice.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const { makeRequest } = require('../../commons');
+
+module.exports = {
+
+    async tick(context) {
+
+        let { changedSince } = await context.stateGet('changedSince') || {};
+
+        if (!changedSince) {
+            // On first tick, fetch only the most recent update to set changedSince
+            changedSince = new Date();
+            await context.stateSet('changedSince', { changedSince });
+        }
+
+        const options = {
+            path: `v3/company/${context.profileInfo.companyId}/cdc?entities=Invoice&changedSince=${changedSince.toISOString()}`,
+            method: 'GET'
+        };
+
+        try {
+            const { data } = await makeRequest({ context, options });
+
+            if (data.CDCResponse[0].QueryResponse[0].Invoice) {
```

Comment:
this is quite fragile

Reply:
From the docs

![image](https://github.com/clientIO/appmixer-connectors/assets/12988096/da68391c-af7b-4e00-b0e9-ff5699fd140a)